services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/geometria/HankeGeometriatServiceImpl.kt

```diff
@@ -62,4 +68,11 @@ open class HankeGeometriatServiceImpl(
         hankeGeometriat?.includeHankeProperties(hanke)
         return hankeGeometriat
     }
+
+    private fun isGeometryNonEmpty(hankeGeometriat: HankeGeometriat): Boolean {
+        // TODO: might check deeper, and if so, there could be some multi-thread unsafe things hidden (though mostly theoretical)...
+        if (hankeGeometriat.featureCollection?.features.isNullOrEmpty()) return false
+        return true
+    }
```

Comment:
Is hankegeometriatId enough for the check? Not sure, but isn't it created only after we have successful insert to database?

Reply:
Not sure. Not really sure if that is enough either. I will happily let someone who knows the geometry side better to improve that function.